browser/profiles/BUILD.gn

```diff
@@ -25,7 +25,6 @@ source_set("profiles") {
 
   deps = [
     "//base",
-    "//brave/browser/gcm_driver",
     "//brave/browser/tor",
```

Comment:
this has the same problem as the other PR. We can't just remove deps, they need to be replaced with the correct dep or they'll get lost